community-membership.md

```diff
@@ -220,6 +219,26 @@ The following apply to the subproject for which one would be an owner.
 - Work with other maintainers to maintain the project's overall health and
   success holistically.
 
+### Becoming a Maintainer
+
+Unless stated otherwise in a SIG charter ratified by the Technical Committee,
+a new maintainer is elected by unanimous vote of the existing maintainers
+of the SIG. After a successful vote, a pull request should be made adding
+the new maintainer to the list of existing maintainers. Before the pull request
+is merged, all existing maintainers and the newly elected maintainer should
+approve the pull request to show their support for the public record. The
+nominee is considered a maintainer after the pull request is merged.
+
+If you feel like you meet the requirements above and are willing to take on the
```

Comment:
> requirements above

I don't think "above" actually lists any requirements to _becoming a maintainer_, it only lists responsibilities of _being_ a maintainer. In Jaeger we had [specific requirements][1] that must be met, where the actual required stats were less important than the fact that they are written down (for a fair process).

Do we want to propose an org-level maintainer stats requirements so that SIGs won't need to define their own (but still have the option to do so)?

[1]: https://github.com/jaegertracing/jaeger/blob/main/GOVERNANCE.md#becoming-a-maintainer

Reply:
https://github.com/open-telemetry/community/blob/main/community-membership.md#requirements-3

Reply:
It's not qualitative, and in fact the only line related to direct code contributions (which I would consider a must for all non-spec repos) is somewhat weak:

> Directly contributed to the subproject through implementation and / or review

Reply:
i think the requirements of the position should be a separate PR from the process itself

Reply:
+1